mcs/class/Mono.Posix/Mono.Unix.Native/Stdlib.cs

```diff
@@ -378,7 +380,11 @@ static XPrintfFunctions ()
 	//
 	public class Stdlib
 	{
+#if NETSTANDARD2_0
+		internal const string LIBC = "c";
```

Comment:
This is really just for .Net Core.  We do have to consider this for the packaging step.  

Maybe to make things clearer I use a different preprocessor such as FORCE_USE_LIBC_NOT_MSVC.

The other option I am thinking of is to push all of these calls into MonoPosixHelpers and not PInvoke libc directly.  If we want to go that way I think that is separate PR as it is large.